packages/nostr/src/EventKind.ts

```diff
@@ -1,4 +1,4 @@
-const enum EventKind {
+enum EventKind {
```

Comment:
`const enum` does not exist in JS and since this is a package now it has to export a JS interface.